core/designsystem/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/designsystem/theme/Type.kt

```diff
@@ -1,28 +1,128 @@
 package io.github.droidkaigi.confsched2023.designsystem.theme
 
 import androidx.compose.material3.Typography
+import androidx.compose.runtime.Composable
 import androidx.compose.ui.text.TextStyle
-import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
+import org.jetbrains.compose.resources.ExperimentalResourceApi
 
 // Set of Material typography styles to start with
-val Typography = Typography(
+@OptIn(ExperimentalResourceApi::class)
+@Composable
+fun typography(): Typography = Typography(
+    displayLarge = TextStyle(
+        fontSize = 57.sp,
+        fontFamily = fontFamilyResource(FontResource.Montserrat).value,
```

Comment:
I don't think we have to call this every text style because we load the font every call. 😅 
We can do it like this

 {
 val montserrat = fontFamilyResource(FontResource.Montserrat)